src/stream_base.cc

```diff
@@ -102,8 +102,7 @@ int StreamBase::Writev(const FunctionCallbackInfo<Value>& args) {
 
   size_t count = chunks->Length() >> 1;
 
-  uv_buf_t bufs_[16];
-  uv_buf_t* bufs = bufs_;
+  MaybeStackBuffer<uv_buf_t> bufs(count);
```

Comment:
Excellent. I suppose the same change should be made for for `udp_wrap.cc` too https://github.com/nodejs/node/pull/8626/commits/a34532265c00bd15eb541a3adbd81fa8caa3faf4?diff=split#diff-b62464f44488c6247346b82a87cbd20aL278